src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { funFacts, FunFact } from "./funfacts"
+import { useState } from "react"
+import "./FunFactPage.scss"
```

Comment:
Semicolons here please! And `React` and `useState` are imported from the same place so should be on the same line
```suggestion
import React, { useState } from "react";
import { funFacts, FunFact } from "./funfacts";
import "./FunFactPage.scss";
```

Reply:
done, thanks